client/index.js

```diff
@@ -1,12 +1,43 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import PictureUnit from './components/PictureUnit';
 
-const App = () => {
-  return (
-    <h1>Hello. I am being rendered</h1>
-  )
-};
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      leftPointer: 0,
+      rightPointer: 2,
+      rawData: this.props.sampleData,
+    }
+  }
 
-ReactDOM.render(<App />, document.getElementById('app'));
+  // componentDidMount() {
+  //   // fetch request to populate data
+  // }
+
+  handleLeftArrowClick() {
+    // slide the picture unit to the right
+  }
+
+  handleRightArrowClick(){
+    // slide the picture unit to the left
+  }
+
+  render() {
+    var unitsToRender = [this.state.leftPointer, this.state.leftPointer + 1, this.state.rightPointer]; 
```

Comment:
If I understand this correctly, I think this is going to be the center image, one to the left of it and one to the right of it, when rendering. In terms of naming scheme, I think there is room for more clarity. Like can you simplify it by saying centerPhoto -1, centerPhoto, centerPhoto+1?

Reply:
Yea good idea. I think I will try to implement this.